pages/using/runtime.md

```diff
@@ -3,16 +3,16 @@
 ## Exposed Ports
 
 Resin.io devices expose all ports by default, meaning you can run applications
-which listen on any port without issue.
+which listen on any port without issue. There is no need to have the docker `EXPOSE` command in your `Dockerfile`.
 
 ## Device URLS
 
-Resin.io currently exposes port 8080 for web forwarding. To enable webforwarding, navigate to a device's actions tab on the resin.io dashboard and enable a public url for this device. Resin will then generate a url for that device.
+Resin.io currently exposes port 80 for web forwarding. To enable webforwarding on a specific device, navigate to the device's **actions** tab on the resin.io dashboard and select the `enable a public url for this device` checkbox. Resin will then generate a web accessable url for the device. The URL will be of the form `<RESIN_DEVICE_UUID>.resindevice.io`, where `<RESIN_DEVICE_UUID.` is the unique ID of the device which you can see on your dashboard.
```

Comment:
You have both `web forwarding` and `webforwarding`, it'd be good to be consistent - personally I prefer `web forwarding`